src/rpc/server.cpp

```diff
@@ -256,13 +256,8 @@ static const CRPCCommand vRPCCommands[] =
 
 CRPCTable::CRPCTable()
 {
-    unsigned int vcidx;
-    for (vcidx = 0; vcidx < (sizeof(vRPCCommands) / sizeof(vRPCCommands[0])); vcidx++)
-    {
-        const CRPCCommand *pcmd;
-
-        pcmd = &vRPCCommands[vcidx];
-        mapCommands[pcmd->name].push_back(pcmd);
+    for (const auto& c : vRPCCommands) {
+        std::cout << appendCommand(c.name, &c) << std::endl;
```

Comment:
Looks like some debug output is left here? Note that unlike the original code, `appendCommand()` is more strict as it doesn't append the command If RPC is running (I guess that's the reason for the cout, you wanted to test If appendCommand returns false in practice).